### DISCUSSION TITLE
Missing color in Conditional UI

### DISCUSSION THREAD

**Discussion by ksl28** [Budibase/budibase#8634] [Help] [answered]
Hi :) 

We have a field group with several text fields below, that are consuming an external data provider.
Most of the text fields are boolean values, and some are integers.
I was hoping to set the value filed to green if the value is true, and red if the value was red - also if the integer was lower than 300 then turn red, otherwise be green.
I found this guide - https://docs.budibase.com/docs/conditions

But im not able to select the Color option for some reason. Any idea to why that is?

![image](https://user-images.githubusercontent.com/53445047/201107606-8f1bb661-1362-4019-bebc-5856e785ae7a.png)

**Comment by melohagan** [comment, marked as answer]
Hey @ksl28 

The reason there is no colour option, is because the text field component has no setting in the settings panel for colour!

You can try setting the Custom CSS, although this will be limited as it's inline style. For example I was able to set the background colour to red:

![Screenshot 2022-11-10 at 15 40 44](https://user-images.githubusercontent.com/101575380/201139418-1bb327f6-a894-463f-815e-0bf68c75733c.png)

You might want to have a look at creating a custom form field for this one: https://docs.budibase.com/docs/custom-component#example-custom-form-field-component

**Reply by ksl28** [reply]
Thanks for the answer :) 
It seems a bit over complicated, to achieve a simple task like this - i guess i will create a feature request for it, and then hope it will be approved :)
